### PR TITLE
docs: add manual CL update instructions

### PR DESCRIPTION
**Pull request by zbwright**
Adding Rob to take a look at the updated info for the updates page. Stephen Augustus to review the manual update information, but he is not a member of this repo.

### CODE REVIEW

File: Documentation/admin/upgrade.md [hunk 1]
```diff
@@ -15,10 +15,30 @@ Please note that the update payload process may affect any or all components in
...
+To provide the most secure operating system, Tectonic automatically updates Container Linux. To manage the update process for Container Linux, rather than allowing it to run automatically, configure the system to allow manual updates.
+
+First, scale the container-linux-update-operator deployment to 0 to pause automated updates.
+* Go to *Workloads > Deployments*, and select the container-linux-update-operator.
+* Set the desired count to 0 Pods, and click *Save*.
+
+Then, mask update-engine to prevent it from downloading or installing updates.
+For more information, see [Disable Automatic Updates Daemon][disable-auto] in the Container Linux reboot strategies guide.
+
+Reset the deployment.
```
Comment (robszumski) [inline Documentation/admin/upgrade.md]: is this telling you to set it back to 0 again? or are you saying scale it back up? back up to what?

File: Documentation/admin/upgrade.md [hunk 2]
```diff
@@ -15,10 +15,30 @@ Please note that the update payload process may affect any or all components in
...
+* Go to *Workloads > Deployments*, and select the container-linux-update-operator.
+* Reset the Pod count, and click *Save*.
+
+Apply CL updates before applying Tectonic updates.
```
Comment (robszumski) [inline Documentation/admin/upgrade.md]: I dont think we use this CL notation anywhere publically

Comment (robszumski) [reply Documentation/admin/upgrade.md]: if this is about updates, should we expand?

**Comment by zbwright** [comment]
we're going to not do this here. I'll close this PR, and open another to update the intro on the page.

**Pull request closed without merging**